source/common/ssl/context_impl.cc

```diff
@@ -866,5 +988,16 @@ void ServerContextImpl::TlsContext::addClientValidationContext(
   }
 }
 
+bool ServerContextImpl::TlsContext::isCipherEnabled(const SSL_CIPHER* cipher) {
+  STACK_OF(SSL_CIPHER)* our_ciphers = SSL_CTX_get_ciphers(ssl_ctx_.get());
+  for (size_t j = 0; j < sk_SSL_CIPHER_num(our_ciphers); j++) {
```

Comment:
You can just write:

```
for (const SSL_CIPHER* our_c : SSL_CTX_get_ciphers(ssl_ctx_.get())) {
  if (SSL_CIPHER_get_id(our_c) == SSL_CIPHER_get_id(cipher)) {
    return true;
  }
}
return false;
```
I was bored one day and made range-for loops work on `STACK_OF(T)`. :-)